[debug] [2025-09-03T18:08:06.227Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-03T18:08:06.764Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-03T18:08:06.764Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-03T18:08:06.764Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-03T18:08:06.766Z] Checked if tokens are valid: false, expires at: 1756809436767
[debug] [2025-09-03T18:08:06.767Z] Checked if tokens are valid: false, expires at: 1756809436767
[debug] [2025-09-03T18:08:06.767Z] > refreshing access token with scopes: []
[debug] [2025-09-03T18:08:06.767Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-03T18:08:06.767Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-03T18:08:06.771Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.11.2) on port 50001
[info] i  dataconnect: downloading dataconnect-emulator-2.11.2... {"metadata":{"emulator":{"name":"dataconnect"},"message":"downloading dataconnect-emulator-2.11.2..."}}
[debug] [2025-09-03T18:08:06.979Z] >>> [apiv2][query] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.11.2 
[debug] [2025-09-03T18:08:06.985Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-09-03T18:08:06.985Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-03T18:08:06.994Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-03T18:08:06.994Z] Checked if tokens are valid: true, expires at: 1756926485985
[debug] [2025-09-03T18:08:06.995Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-03T18:08:07.300Z] <<< [apiv2][status] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.11.2 200
[debug] [2025-09-03T18:08:07.301Z] <<< [apiv2][body] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.11.2 [stream]
[debug] [2025-09-03T18:08:07.654Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-03T18:08:07.654Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-03T18:08:14.098Z] 'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0903 20:08:14.095296   41113 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"'fdc build' failed with error: Unable to build your Data Connect schema and connectors (exit code 1): E0903 20:08:14.095296   41113 cmd.go:43] Error loading data connect config: could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:08:14.098Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-03T18:08:14.098Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-03T18:08:14.099Z] Starting Data Connect Emulator with command {"binary":"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-03T18:08:14.669Z] E0903 20:08:14.666234   41115 dev.go:79] Failed to load initial config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:08:14.666234   41115 dev.go:79] Failed to load initial config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:08:14.677Z] E0903 20:08:14.673576   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:08:14.673576   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:08:14.701Z] I0903 20:08:14.697739   41115 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:08:14.697739   41115 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[error] [Firebase Plugin] error:  {}
[debug] [2025-09-03T18:08:24.789Z] E0903 20:08:24.774256   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:08:24.774256   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:08:34.693Z] E0903 20:08:34.691825   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:08:34.691825   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:08:44.685Z] E0903 20:08:44.681195   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:08:44.681195   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:08:54.691Z] E0903 20:08:54.678940   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:08:54.678940   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:09:04.717Z] E0903 20:09:04.705317   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:09:04.705317   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:09:14.678Z] E0903 20:09:14.676944   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:09:14.676944   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:09:18.269Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: undefined
[debug] [2025-09-03T18:09:18.269Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-03T18:09:18.270Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-03T18:09:18.270Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-03T18:09:18.270Z] Checked if tokens are valid: true, expires at: 1756926485985
[debug] [2025-09-03T18:09:18.270Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-03T18:09:18.271Z] Checked if tokens are valid: true, expires at: 1756926485985
[debug] [2025-09-03T18:09:18.271Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-03T18:09:18.277Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: undefined
[debug] [2025-09-03T18:09:18.278Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-03T18:09:18.278Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-03T18:09:18.278Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-03T18:09:18.278Z] Checked if tokens are valid: true, expires at: 1756926485985
[debug] [2025-09-03T18:09:18.278Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-03T18:09:18.278Z] Checked if tokens are valid: true, expires at: 1756926485985
[debug] [2025-09-03T18:09:18.278Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-03T18:09:18.884Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-03T18:09:18.884Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-03T18:09:18.894Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-03T18:09:18.894Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-03T18:09:24.807Z] E0903 20:09:24.677599   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:09:24.677599   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:09:34.679Z] E0903 20:09:34.676612   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:09:34.676612   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:09:44.683Z] E0903 20:09:44.681313   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:09:44.681313   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:09:54.680Z] E0903 20:09:54.679003   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:09:54.679003   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:10:04.682Z] E0903 20:10:04.679282   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:10:04.679282   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:10:14.681Z] E0903 20:10:14.677428   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:10:14.677493   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:10:14.677428   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:10:14.677493   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:10:24.681Z] E0903 20:10:24.680241   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:10:24.680241   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:10:34.687Z] E0903 20:10:34.686892   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:10:34.686892   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:10:44.680Z] E0903 20:10:44.679169   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:10:44.679169   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:10:54.685Z] E0903 20:10:54.681616   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:10:54.681616   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:11:04.684Z] E0903 20:11:04.682839   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:11:04.682839   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:11:14.684Z] E0903 20:11:14.679239   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:11:14.679239   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:11:24.683Z] E0903 20:11:24.681966   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:11:24.681966   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:11:34.687Z] E0903 20:11:34.684938   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:11:34.684938   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:11:44.690Z] E0903 20:11:44.685478   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:11:44.685478   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:11:54.685Z] E0903 20:11:54.680317   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:11:54.680317   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:12:04.988Z] E0903 20:12:04.678914   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:12:04.678914   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:12:14.682Z] E0903 20:12:14.678609   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:12:14.678675   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:12:14.678609   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:12:14.678675   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:12:25.030Z] E0903 20:12:24.678541   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:12:24.678541   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:12:34.685Z] E0903 20:12:34.680840   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:12:34.680840   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:12:44.787Z] E0903 20:12:44.678761   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:12:44.678761   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:12:54.684Z] E0903 20:12:54.679417   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:12:54.679417   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:13:04.683Z] E0903 20:13:04.683457   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:13:04.683457   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:13:14.686Z] E0903 20:13:14.684020   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:13:14.684020   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:13:24.680Z] E0903 20:13:24.678850   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:13:24.678850   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:13:34.682Z] E0903 20:13:34.681139   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:13:34.681139   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:13:44.682Z] E0903 20:13:44.680193   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:13:44.680193   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:13:54.684Z] E0903 20:13:54.680581   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:13:54.680581   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:14:04.689Z] E0903 20:14:04.679630   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:14:04.679630   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:14:14.694Z] E0903 20:14:14.682468   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:14:14.682527   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:14:14.682468   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:14:14.682527   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:14:24.686Z] E0903 20:14:24.685490   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:14:24.685490   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:14:34.695Z] E0903 20:14:34.682025   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:14:34.682025   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:14:44.693Z] E0903 20:14:44.683442   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:14:44.683442   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:14:54.686Z] E0903 20:14:54.682837   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:14:54.682837   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:15:04.689Z] E0903 20:15:04.684457   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:15:04.684457   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:15:14.682Z] E0903 20:15:14.680112   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:15:14.680112   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:15:24.682Z] E0903 20:15:24.680633   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:15:24.680633   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:15:34.706Z] E0903 20:15:34.695402   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:15:34.695402   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:15:44.684Z] E0903 20:15:44.683242   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:15:44.683242   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:15:54.686Z] E0903 20:15:54.680732   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:15:54.680732   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:16:04.683Z] E0903 20:16:04.682203   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:16:04.682203   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:16:15.050Z] E0903 20:16:14.680664   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:16:14.680724   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:16:14.680664   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:16:14.680724   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:16:24.682Z] E0903 20:16:24.680633   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:16:24.680633   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:16:34.686Z] E0903 20:16:34.681171   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:16:34.681171   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:16:44.688Z] E0903 20:16:44.684321   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:16:44.684321   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:16:54.684Z] E0903 20:16:54.683664   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:16:54.683664   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:17:04.685Z] E0903 20:17:04.682258   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:17:04.682258   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:17:14.689Z] E0903 20:17:14.687669   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:17:14.687669   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:17:24.687Z] E0903 20:17:24.685236   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:17:24.685236   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:17:34.686Z] E0903 20:17:34.682145   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:17:34.682145   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:17:44.683Z] E0903 20:17:44.683040   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:17:44.683040   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:17:54.685Z] E0903 20:17:54.682872   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:17:54.682872   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:18:04.740Z] E0903 20:18:04.714193   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:18:04.714193   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:18:14.692Z] E0903 20:18:14.683476   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:18:14.683535   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:18:14.683476   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:18:14.683535   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:18:24.689Z] E0903 20:18:24.683367   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:18:24.683367   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:18:34.682Z] E0903 20:18:34.682278   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:18:34.682278   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:18:44.690Z] E0903 20:18:44.685479   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:18:44.685479   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:18:54.686Z] E0903 20:18:54.685686   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:18:54.685686   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:19:04.688Z] E0903 20:19:04.682503   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:19:04.682503   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:19:14.713Z] E0903 20:19:14.684051   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:19:14.684051   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:19:25.418Z] E0903 20:19:24.682381   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:19:24.682381   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:19:34.687Z] E0903 20:19:34.682805   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:19:34.682805   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:19:44.689Z] E0903 20:19:44.682533   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:19:44.682533   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:19:54.688Z] E0903 20:19:54.683066   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:19:54.683066   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:20:04.692Z] E0903 20:20:04.683863   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:20:04.683863   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:20:14.686Z] E0903 20:20:14.684506   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:20:14.684561   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:20:14.684506   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:20:14.684561   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:20:24.684Z] E0903 20:20:24.683934   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:20:24.683934   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:20:34.687Z] E0903 20:20:34.684723   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:20:34.684723   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:20:44.689Z] E0903 20:20:44.685610   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:20:44.685610   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:20:54.687Z] E0903 20:20:54.684982   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:20:54.684982   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:21:04.700Z] E0903 20:21:04.691399   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:21:04.691399   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:21:14.685Z] E0903 20:21:14.683911   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:21:14.683911   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:21:24.687Z] E0903 20:21:24.684241   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:21:24.684241   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:21:34.687Z] E0903 20:21:34.684323   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:21:34.684323   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:21:44.693Z] E0903 20:21:44.683818   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:21:44.683818   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:21:54.688Z] E0903 20:21:54.687687   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:21:54.687687   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:22:04.698Z] E0903 20:22:04.686147   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:22:04.686147   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:22:14.688Z] E0903 20:22:14.684097   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:22:14.684351   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:22:14.684097   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:22:14.684351   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:22:25.197Z] E0903 20:22:24.684025   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:22:24.684025   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:22:34.689Z] E0903 20:22:34.685240   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:22:34.685240   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:22:44.691Z] E0903 20:22:44.685631   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:22:44.685631   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:22:54.701Z] E0903 20:22:54.692763   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:22:54.692763   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:23:04.692Z] E0903 20:23:04.688264   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:23:04.688264   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:23:14.690Z] E0903 20:23:14.685357   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:23:14.685357   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:23:24.694Z] E0903 20:23:24.687218   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:23:24.687218   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:23:34.703Z] E0903 20:23:34.691314   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:23:34.691314   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:23:44.690Z] E0903 20:23:44.689206   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:23:44.689206   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:23:54.687Z] E0903 20:23:54.686450   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:23:54.686450   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:24:04.686Z] E0903 20:24:04.686051   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:24:04.686051   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:24:14.691Z] E0903 20:24:14.685305   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:24:14.686021   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:24:14.685305   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:24:14.686021   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:24:25.097Z] E0903 20:24:24.684987   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:24:24.684987   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:24:34.690Z] E0903 20:24:34.686079   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:24:34.686079   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:24:44.695Z] E0903 20:24:44.687181   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:24:44.687181   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:24:54.700Z] E0903 20:24:54.700250   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:24:54.700250   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:25:04.691Z] E0903 20:25:04.686962   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:25:04.686962   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:25:14.691Z] E0903 20:25:14.687301   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:25:14.687301   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:25:24.690Z] E0903 20:25:24.685693   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:25:24.685693   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:25:34.697Z] E0903 20:25:34.687022   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:25:34.687022   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:25:44.689Z] E0903 20:25:44.686995   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:25:44.686995   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:25:54.687Z] E0903 20:25:54.685373   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:25:54.685373   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:26:04.716Z] E0903 20:26:04.712414   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:26:04.712414   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:26:14.691Z] E0903 20:26:14.688436   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
E0903 20:26:14.688496   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:26:14.688436   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\nE0903 20:26:14.688496   41115 watch.go:102] watchAllDir unable to walk dir `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`: lstat /Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect: no such file or directory\n"}}
[debug] [2025-09-03T18:26:24.695Z] E0903 20:26:24.691080   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:26:24.691080   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:26:34.694Z] E0903 20:26:34.689224   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:26:34.689224   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:26:44.691Z] E0903 20:26:44.687027   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:26:44.687027   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:26:54.696Z] E0903 20:26:54.686665   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:26:54.686665   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:27:04.700Z] E0903 20:27:04.690612   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:27:04.690612   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:27:14.716Z] E0903 20:27:14.704191   41115 watch.go:31] Failed to load config from "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"E0903 20:27:14.704191   41115 watch.go:31] Failed to load config from \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect\": could not find dataconnect.yaml in `/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect`\n"}}
[debug] [2025-09-03T18:27:24.729Z] I0903 20:27:24.724732   41115 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:27:24.724732   41115 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-03T18:27:25.398Z] I0903 20:27:25.392964   41115 control.go:94] [/emulator/vulugo-v100 0dad] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/mutations.gql [1243B] example/queries.gql [1801B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:27:25.392964   41115 control.go:94] [/emulator/vulugo-v100 0dad] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/mutations.gql [1243B] example/queries.gql [1801B] \n"}}
[debug] [2025-09-03T18:27:25.582Z] I0903 20:27:25.573358   41115 codegen.go:82] [connector "example" javascriptSdk] Generating sources into "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./src/dataconnect-generated"
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:27:25.573358   41115 codegen.go:82] [connector \"example\" javascriptSdk] Generating sources into \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./src/dataconnect-generated\"\n"}}
[debug] [2025-09-03T18:27:25.660Z] I0903 20:27:25.651044   41115 collector.go:107] schema extensions wrote into "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect/.dataconnect/schema"
Generated sources: main/mutation.gql [13679B] main/implicit.gql [1261B] main/input.gql [28173B] prelude.gql [73251B] main/relation.gql [13742B] main/query.gql [5099B] 
I0903 20:27:25.655396   41115 collector.go:107] connector "example" javascriptSdk wrote into "/Users/omid/VuluGO v6/VuluGO-v.1.0.0./src/dataconnect-generated"
Generated sources: index.d.ts [8025B] react/index.d.ts [3961B] react/esm/package.json [17B] react/README.md [48299B] esm/index.esm.js [3564B] esm/package.json [17B] index.cjs.js [4054B] package.json [852B] README.md [40931B] react/package.json [460B] react/index.cjs.js [3439B] react/esm/index.esm.js [3269B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:27:25.651044   41115 collector.go:107] schema extensions wrote into \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./dataconnect/.dataconnect/schema\"\nGenerated sources: main/mutation.gql [13679B] main/implicit.gql [1261B] main/input.gql [28173B] prelude.gql [73251B] main/relation.gql [13742B] main/query.gql [5099B] \nI0903 20:27:25.655396   41115 collector.go:107] connector \"example\" javascriptSdk wrote into \"/Users/omid/VuluGO v6/VuluGO-v.1.0.0./src/dataconnect-generated\"\nGenerated sources: index.d.ts [8025B] react/index.d.ts [3961B] react/esm/package.json [17B] react/README.md [48299B] esm/index.esm.js [3564B] esm/package.json [17B] index.cjs.js [4054B] package.json [852B] README.md [40931B] react/package.json [460B] react/index.cjs.js [3439B] react/esm/index.esm.js [3269B] \n"}}
[debug] [2025-09-03T18:27:25.671Z] I0903 20:27:25.668172   41115 installsdk.go:86] Running command: /usr/local/bin/npm install file:src/dataconnect-generated
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:27:25.668172   41115 installsdk.go:86] Running command: /usr/local/bin/npm install file:src/dataconnect-generated\n"}}
[debug] [2025-09-03T18:28:21.466Z] I0903 20:28:21.147935   41115 load.go:109] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0903 20:28:21.147935   41115 load.go:109] Finished reloading\n"}}
[debug] [2025-09-05T03:37:06.984Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-05T03:37:07.450Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-05T03:37:07.450Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-05T03:37:07.450Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-05T03:37:07.452Z] Checked if tokens are valid: true, expires at: 1757046974194
[debug] [2025-09-05T03:37:07.452Z] Checked if tokens are valid: true, expires at: 1757046974194
[debug] [2025-09-05T03:37:07.452Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-05T03:37:07.452Z] Checked if tokens are valid: true, expires at: 1757046974194
[debug] [2025-09-05T03:37:07.453Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.11.2) on port 50002
[info] i  dataconnect: downloading dataconnect-emulator-2.11.2... {"metadata":{"emulator":{"name":"dataconnect"},"message":"downloading dataconnect-emulator-2.11.2..."}}
[debug] [2025-09-05T03:37:07.602Z] >>> [apiv2][query] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.11.2 
[debug] [2025-09-05T03:37:07.920Z] <<< [apiv2][status] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.11.2 200
[debug] [2025-09-05T03:37:07.921Z] <<< [apiv2][body] GET https://storage.googleapis.com/firemat-preview-drop/emulator/dataconnect-emulator-macos-v2.11.2 [stream]
[debug] [2025-09-05T03:37:08.226Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-05T03:37:08.226Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[info] i  dataconnect: Removing outdated emulator files: dataconnect-emulator-2.12.0 {"metadata":{"emulator":{"name":"dataconnect"},"message":"Removing outdated emulator files: dataconnect-emulator-2.12.0"}}
[debug] [2025-09-05T03:37:11.814Z] I0905 05:37:11.797400   59648 control.go:94] [/emulator/vulugo-v100 5354] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/mutations.gql [1243B] example/queries.gql [1801B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:11.797400   59648 control.go:94] [/emulator/vulugo-v100 5354] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/mutations.gql [1243B] example/queries.gql [1801B] \n"}}
[debug] [2025-09-05T03:37:11.815Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-05T03:37:11.815Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-05T03:37:11.815Z] Starting Data Connect Emulator with command {"binary":"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2","args":["--logtostderr","-v=2","dev","--listen=localhost:50002","--config_dir=/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50002\",\"--config_dir=/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-05T03:37:11.979Z] I0905 05:37:11.978419   59658 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:11.978419   59658 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-05T03:37:12.056Z] I0905 05:37:12.056090   59658 control.go:94] [/emulator/vulugo-v100 5e2f] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/mutations.gql [1243B] example/queries.gql [1801B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.056090   59658 control.go:94] [/emulator/vulugo-v100 5e2f] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/mutations.gql [1243B] example/queries.gql [1801B] \n"}}
[debug] [2025-09-05T03:37:12.085Z] I0905 05:37:12.085033   59658 codegen.go:82] [connector "example" javascriptSdk] Generating sources into "/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./src/dataconnect-generated"
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.085033   59658 codegen.go:82] [connector \"example\" javascriptSdk] Generating sources into \"/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./src/dataconnect-generated\"\n"}}
[debug] [2025-09-05T03:37:12.100Z] I0905 05:37:12.100572   59658 collector.go:107] schema extensions wrote into "/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/input.gql [28173B] main/relation.gql [13742B] main/query.gql [5099B] main/mutation.gql [13679B] main/implicit.gql [1261B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.100572   59658 collector.go:107] schema extensions wrote into \"/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/input.gql [28173B] main/relation.gql [13742B] main/query.gql [5099B] main/mutation.gql [13679B] main/implicit.gql [1261B] \n"}}
[debug] [2025-09-05T03:37:12.101Z] I0905 05:37:12.101610   59658 collector.go:107] connector "example" javascriptSdk wrote into "/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./src/dataconnect-generated"
Generated sources: package.json [852B] react/index.d.ts [3961B] react/esm/package.json [17B] react/README.md [48299B] esm/index.esm.js [3564B] esm/package.json [17B] README.md [40931B] react/package.json [460B] react/index.cjs.js [3439B] react/esm/index.esm.js [3269B] index.d.ts [8025B] index.cjs.js [4054B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.101610   59658 collector.go:107] connector \"example\" javascriptSdk wrote into \"/Users/omid/Vulu v2.0/VuluGO-v.1.0.0./src/dataconnect-generated\"\nGenerated sources: package.json [852B] react/index.d.ts [3961B] react/esm/package.json [17B] react/README.md [48299B] esm/index.esm.js [3564B] esm/package.json [17B] README.md [40931B] react/package.json [460B] react/index.cjs.js [3439B] react/esm/index.esm.js [3269B] index.d.ts [8025B] index.cjs.js [4054B] \n"}}
[debug] [2025-09-05T03:37:12.105Z] I0905 05:37:12.105288   59658 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.105288   59658 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation\n"}}
[debug] [2025-09-05T03:37:12.105Z] I0905 05:37:12.105428   59658 load.go:109] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.105428   59658 load.go:109] Finished reloading\n"}}
[debug] [2025-09-05T03:37:12.112Z] I0905 05:37:12.111713   59658 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50002
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.111713   59658 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50002\n"}}
[debug] [2025-09-05T03:37:12.392Z] I0905 05:37:12.391924   59658 engine.go:171] [/emulator/vulugo-v100 5e2f] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 05:37:12.391924   59658 engine.go:171] [/emulator/vulugo-v100 5e2f] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-05T17:30:23.077Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-05T17:30:23.721Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-05T17:30:23.722Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-05T17:30:23.722Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-05T17:30:23.724Z] Checked if tokens are valid: true, expires at: 1757096977750
[debug] [2025-09-05T17:30:23.724Z] Checked if tokens are valid: true, expires at: 1757096977750
[debug] [2025-09-05T17:30:23.724Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-05T17:30:23.724Z] Checked if tokens are valid: true, expires at: 1757096977750
[debug] [2025-09-05T17:30:23.725Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-05T17:30:23.729Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.11.2) on port 50001
[debug] [2025-09-05T17:30:26.149Z] I0905 19:30:26.131535   21533 control.go:94] [/emulator/vulugo-v100 a262] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/mutations.gql [1243B] example/queries.gql [1801B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.131535   21533 control.go:94] [/emulator/vulugo-v100 a262] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/mutations.gql [1243B] example/queries.gql [1801B] \n"}}
[debug] [2025-09-05T17:30:26.149Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-05T17:30:26.149Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-05T17:30:26.149Z] Starting Data Connect Emulator with command {"binary":"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/Users/omid/Downloads/Vulu v2.1/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/Users/omid/Downloads/Vulu v2.1/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-05T17:30:26.641Z] I0905 19:30:26.546339   21534 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.546339   21534 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-05T17:30:26.788Z] I0905 19:30:26.782686   21534 control.go:94] [/emulator/vulugo-v100 b01f] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/mutations.gql [1243B] example/queries.gql [1801B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.782686   21534 control.go:94] [/emulator/vulugo-v100 b01f] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/mutations.gql [1243B] example/queries.gql [1801B] \n"}}
[debug] [2025-09-05T17:30:26.908Z] I0905 19:30:26.908851   21534 codegen.go:82] [connector "example" javascriptSdk] Generating sources into "/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated"
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.908851   21534 codegen.go:82] [connector \"example\" javascriptSdk] Generating sources into \"/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated\"\n"}}
[debug] [2025-09-05T17:30:26.936Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-05T17:30:26.936Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-05T17:30:26.939Z] I0905 19:30:26.938106   21534 collector.go:107] schema extensions wrote into "/Users/omid/Downloads/Vulu v2.1/dataconnect/.dataconnect/schema"
Generated sources: prelude.gql [73251B] main/input.gql [28173B] main/relation.gql [13742B] main/query.gql [5099B] main/mutation.gql [13679B] main/implicit.gql [1261B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.938106   21534 collector.go:107] schema extensions wrote into \"/Users/omid/Downloads/Vulu v2.1/dataconnect/.dataconnect/schema\"\nGenerated sources: prelude.gql [73251B] main/input.gql [28173B] main/relation.gql [13742B] main/query.gql [5099B] main/mutation.gql [13679B] main/implicit.gql [1261B] \n"}}
[debug] [2025-09-05T17:30:26.939Z] I0905 19:30:26.939267   21534 collector.go:107] connector "example" javascriptSdk wrote into "/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated"
Generated sources: index.d.ts [8025B] esm/index.esm.js [3564B] index.cjs.js [4054B] README.md [40931B] react/index.d.ts [3961B] react/package.json [460B] react/esm/index.esm.js [3269B] esm/package.json [17B] package.json [852B] react/esm/package.json [17B] react/index.cjs.js [3439B] react/README.md [48299B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.939267   21534 collector.go:107] connector \"example\" javascriptSdk wrote into \"/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated\"\nGenerated sources: index.d.ts [8025B] esm/index.esm.js [3564B] index.cjs.js [4054B] README.md [40931B] react/index.d.ts [3961B] react/package.json [460B] react/esm/index.esm.js [3269B] esm/package.json [17B] package.json [852B] react/esm/package.json [17B] react/index.cjs.js [3439B] react/README.md [48299B] \n"}}
[debug] [2025-09-05T17:30:26.944Z] I0905 19:30:26.944137   21534 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.944137   21534 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation\n"}}
[debug] [2025-09-05T17:30:26.944Z] I0905 19:30:26.944521   21534 load.go:109] Finished reloading
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.944521   21534 load.go:109] Finished reloading\n"}}
[debug] [2025-09-05T17:30:26.951Z] I0905 19:30:26.951499   21534 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:26.951499   21534 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-05T17:30:27.338Z] I0905 19:30:27.338250   21534 engine.go:171] [/emulator/vulugo-v100 b01f] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:30:27.338250   21534 engine.go:171] [/emulator/vulugo-v100 b01f] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[info] [Firebase Plugin] Stopping Data Connect toolkit
[warn] ⚠  Data Connect Emulator has exited upon receiving signal: SIGINT 
[debug] [2025-09-05T17:31:15.909Z] [Firebase Plugin] Activating Firebase extension.
[debug] [2025-09-05T17:31:16.324Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-05T17:31:16.324Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-05T17:31:16.324Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-05T17:31:16.326Z] Checked if tokens are valid: true, expires at: 1757096977750
[debug] [2025-09-05T17:31:16.326Z] Checked if tokens are valid: true, expires at: 1757096977750
[debug] [2025-09-05T17:31:16.326Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-05T17:31:16.326Z] Checked if tokens are valid: true, expires at: 1757096977750
[debug] [2025-09-05T17:31:16.327Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-05T17:31:16.331Z] [Firebase Plugin] VSCode notification server listening on port 40001
[info] [Firebase Plugin] Starting Data Connect toolkit (version 2.11.2) on port 50001
[debug] [2025-09-05T17:31:18.061Z] I0905 19:31:18.039080   22594 control.go:94] [/emulator/vulugo-v100 097f] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/queries.gql [1801B] example/mutations.gql [1243B] 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:31:18.039080   22594 control.go:94] [/emulator/vulugo-v100 097f] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/queries.gql [1801B] example/mutations.gql [1243B] \n"}}
[debug] [2025-09-05T17:31:18.062Z] Ignoring unsupported arg: auto_download {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: auto_download"}}
[debug] [2025-09-05T17:31:18.062Z] Ignoring unsupported arg: port {"metadata":{"emulator":{"name":"dataconnect"},"message":"Ignoring unsupported arg: port"}}
[debug] [2025-09-05T17:31:18.062Z] Starting Data Connect Emulator with command {"binary":"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2","args":["--logtostderr","-v=2","dev","--listen=localhost:50001","--config_dir=/Users/omid/Downloads/Vulu v2.1/dataconnect","--enable_output_schema_extensions=true","--enable_output_generated_sdk=true"],"optionalArgs":["listen","config_dir","enable_output_schema_extensions","enable_output_generated_sdk"],"joinArgs":true,"shell":false,"port":9399} {"metadata":{"emulator":{"name":"dataconnect"},"message":"Starting Data Connect Emulator with command {\"binary\":\"/Users/omid/.cache/firebase/emulators/dataconnect-emulator-2.11.2\",\"args\":[\"--logtostderr\",\"-v=2\",\"dev\",\"--listen=localhost:50001\",\"--config_dir=/Users/omid/Downloads/Vulu v2.1/dataconnect\",\"--enable_output_schema_extensions=true\",\"--enable_output_generated_sdk=true\"],\"optionalArgs\":[\"listen\",\"config_dir\",\"enable_output_schema_extensions\",\"enable_output_generated_sdk\"],\"joinArgs\":true,\"shell\":false,\"port\":9399}"}}
[info] i  dataconnect: Data Connect Emulator logging to dataconnect-debug.log {"metadata":{"emulator":{"name":"dataconnect"},"message":"Data Connect Emulator logging to dataconnect-debug.log"}}
[debug] [2025-09-05T17:31:18.368Z] I0905 19:31:18.323238   22595 load.go:31] Reloading schema and connectors...
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:31:18.323238   22595 load.go:31] Reloading schema and connectors...\n"}}
[debug] [2025-09-05T17:31:18.556Z] I0905 19:31:18.441696   22595 control.go:94] [/emulator/vulugo-v100 22ea] UpdateResources(): done
Schema: sources: schema/schema.gql [1992B] 
Connector "example": sources: example/mutations.gql [1243B] example/queries.gql [1801B] 
I0905 19:31:18.483237   22595 codegen.go:82] [connector "example" javascriptSdk] Generating sources into "/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated"
I0905 19:31:18.504681   22595 collector.go:107] schema extensions wrote into "/Users/omid/Downloads/Vulu v2.1/dataconnect/.dataconnect/schema"
Generated sources: main/input.gql [28173B] main/relation.gql [13742B] main/query.gql [5099B] main/mutation.gql [13679B] prelude.gql [73251B] main/implicit.gql [1261B] 
I0905 19:31:18.506270   22595 collector.go:107] connector "example" javascriptSdk wrote into "/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated"
Generated sources: index.d.ts [8025B] esm/index.esm.js [3564B] esm/package.json [17B] index.cjs.js [4054B] README.md [40931B] react/package.json [460B] react/index.cjs.js [3439B] react/esm/index.esm.js [3269B] package.json [852B] react/index.d.ts [3961B] react/esm/package.json [17B] react/README.md [48299B] 
I0905 19:31:18.511377   22595 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation
I0905 19:31:18.511594   22595 load.go:109] Finished reloading
I0905 19:31:18.517383   22595 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:31:18.441696   22595 control.go:94] [/emulator/vulugo-v100 22ea] UpdateResources(): done\nSchema: sources: schema/schema.gql [1992B] \nConnector \"example\": sources: example/mutations.gql [1243B] example/queries.gql [1801B] \nI0905 19:31:18.483237   22595 codegen.go:82] [connector \"example\" javascriptSdk] Generating sources into \"/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated\"\nI0905 19:31:18.504681   22595 collector.go:107] schema extensions wrote into \"/Users/omid/Downloads/Vulu v2.1/dataconnect/.dataconnect/schema\"\nGenerated sources: main/input.gql [28173B] main/relation.gql [13742B] main/query.gql [5099B] main/mutation.gql [13679B] prelude.gql [73251B] main/implicit.gql [1261B] \nI0905 19:31:18.506270   22595 collector.go:107] connector \"example\" javascriptSdk wrote into \"/Users/omid/Downloads/Vulu v2.1/src/dataconnect-generated\"\nGenerated sources: index.d.ts [8025B] esm/index.esm.js [3564B] esm/package.json [17B] index.cjs.js [4054B] README.md [40931B] react/package.json [460B] react/index.cjs.js [3439B] react/esm/index.esm.js [3269B] package.json [852B] react/index.d.ts [3961B] react/esm/package.json [17B] react/README.md [48299B] \nI0905 19:31:18.511377   22595 installsdk.go:55] Package @dataconnect/generated is already installed. Skipping installation\nI0905 19:31:18.511594   22595 load.go:109] Finished reloading\nI0905 19:31:18.517383   22595 dev.go:102] Listening on address (HTTP + gRPC): 127.0.0.1:50001\n"}}
[debug] [2025-09-05T17:31:18.723Z] I0905 19:31:18.720024   22595 engine.go:171] [/emulator/vulugo-v100 22ea] ExecuteGraphqlRead IntrospectionQuery: succeeded. 
 {"metadata":{"emulator":{"name":"dataconnect"},"message":"I0905 19:31:18.720024   22595 engine.go:171] [/emulator/vulugo-v100 22ea] ExecuteGraphqlRead IntrospectionQuery: succeeded. \n"}}
[debug] [2025-09-05T17:31:18.762Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-05T17:31:18.763Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-05T21:47:08.007Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: undefined
[debug] [2025-09-05T21:47:08.007Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-05T21:47:08.008Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-05T21:47:08.008Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-05T21:47:08.008Z] Checked if tokens are valid: false, expires at: 1757096977750
[debug] [2025-09-05T21:47:08.008Z] Checked if tokens are valid: false, expires at: 1757096977750
[debug] [2025-09-05T21:47:08.008Z] > refreshing access token with scopes: []
[debug] [2025-09-05T21:47:08.009Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-05T21:47:08.009Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-05T21:47:08.011Z] [Firebase Plugin] Value of process.env.MONOSPACE_ENV: undefined
[debug] [2025-09-05T21:47:08.011Z] [Firebase Plugin] requireAuthWrapper
[debug] [2025-09-05T21:47:08.011Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-05T21:47:08.012Z] > authorizing via signed-in user (vulu.studio@gmail.com)
[debug] [2025-09-05T21:47:08.012Z] Checked if tokens are valid: false, expires at: 1757096977750
[debug] [2025-09-05T21:47:08.012Z] Checked if tokens are valid: false, expires at: 1757096977750
[debug] [2025-09-05T21:47:08.012Z] > refreshing access token with scopes: []
[debug] [2025-09-05T21:47:08.012Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2025-09-05T21:47:08.012Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-05T21:47:08.178Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-09-05T21:47:08.178Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-05T21:47:08.191Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-05T21:47:08.192Z] Checked if tokens are valid: true, expires at: 1757112427178
[debug] [2025-09-05T21:47:08.194Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-05T21:47:08.199Z] <<< [apiv2][status] POST https://www.googleapis.com/oauth2/v3/token 200
[debug] [2025-09-05T21:47:08.199Z] <<< [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2025-09-05T21:47:08.207Z] [Firebase Plugin] User found:  vulu.studio@gmail.com
[info] [Firebase Plugin] (Core:Project) New user detected, fetching projects
[debug] [2025-09-05T21:47:08.208Z] Checked if tokens are valid: true, expires at: 1757112427199
[debug] [2025-09-05T21:47:08.208Z] >>> [apiv2][query] GET https://firebase.googleapis.com/v1beta1/projects pageSize=1000
[debug] [2025-09-05T21:47:08.919Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-05T21:47:08.919Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
[debug] [2025-09-05T21:47:08.940Z] <<< [apiv2][status] GET https://firebase.googleapis.com/v1beta1/projects 200
[debug] [2025-09-05T21:47:08.940Z] <<< [apiv2][body] GET https://firebase.googleapis.com/v1beta1/projects [omitted]
